### DISCUSSION TITLE
Why is there so many executable files ?

### DISCUSSION THREAD

**Discussion by givememymilk** [wezterm/wezterm#4228] [Q&A] [answered]
Whether from source, generic tarball or from archlinux's repo, I notice that installing wezterm always give me 4 seperate executable files which is open-wezterm-here, wezterm, wezterm-gui and wezterm-mux-server. Can someone explain why this is the case?

**Comment by wez** [comment, marked as answer]
Each executable has a different function.
https://github.com/wez/wezterm/blob/main/README-DISTRO-MAINTAINER.md#binaries
